letsencrypt-auto-source/Dockerfile.centos6

```diff
@@ -0,0 +1,32 @@
+# For running tests, build a docker image with a passwordless sudo and a trust
+# store we can manipulate.
+
+FROM centos:6
+
+RUN rpm -ivh http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
```

Comment:
My comment is based on what I did back then (which might be wrong), so alright, I'll try that :)